src/broadcast/helpers.js

```diff
@@ -1,31 +1,32 @@
-import api from "../api";
-
-const defaultWeight = 1;
+import api from '../api';
 
 exports = module.exports = steemBroadcast => {
   steemBroadcast.addAccountAuth = (
     activeWif,
     username,
     authorizedUsername,
-    role = "posting",
+    role = 'posting',
+    weight = 1,
```

Comment:
If the account has a `weight_threshold` of 2 then the newly added key will not be able to broadcast alone if the weight is not set to 2. I can change the default weight to use the same value as `weight_threshold`.